src/python/pants/backend/python/rules/python_test_runner.py

```diff
@@ -21,7 +23,27 @@
 from pants.engine.selectors import Get
 from pants.option.global_options import GlobalOptions
 from pants.rules.core.strip_source_root import SourceRootStrippedSources
-from pants.rules.core.test import TestDebugResult, TestResult, TestTarget
+from pants.rules.core.test import TestDebugResult, TestOptions, TestResult, TestTarget
+
+
+DEFAULT_COVERAGE_CONFIG = dedent(f"""
+  [run]
+  branch = True
+  timid = False
+  relative_files = True
+  """)
```

Comment:
The first two were hardcoded in V1, so just maintaining the existing interface, we can likely make these configurable though.

 The third one is nonnegotiable and *must* be there in order for coverage to work at all in a remote process execution. (Without this setting it will save absolute paths and fail when you attempt to merge the data and those paths do not exist.)

Reply:
Adding additional coverage config via the command line can be made possible in the future, should anyone want it.